### DISCUSSION TITLE
I can't place any of the stones!

### DISCUSSION THREAD

**Discussion by M708591** [Prodigy-Hacking/ProdigyMathGameHacking#953] [(B) Bug Discussion]
Ever since I got the hack I haven't been able to place any of the stones except for the Earth Stone. I had already placed the Earth Stone before I got the hack and I completed the earth tower task. But, keeps on showing an exclamation mark telling me to place one of the stones or something. When I click on it, it says that Gale has to fix the pedestal for this keystone. Also, I had already gotten all of the stones and I don't know why it's showing an exclamation mark on the pedestal.

![Screenshot 2021-05-15 4 51 29 PM](https://user-images.githubusercontent.com/83991714/118381311-91c50300-b59e-11eb-84f5-a1a8977ea331.png)

**Comment by LeoBadeaux** [comment]
Prodigy hasn't finished the rest of the stones, so that's why it won't work. When prodigy finishes programming in the stones it will work.
